lib/Github/Api/ApiInterface.php

```diff
@@ -3,13 +3,19 @@
 namespace Github\Api;
 
 /**
- * Api interface.
- *
  * @author Joseph Bielawski <stloyd@gmail.com>
+ * @author Graham Campbell <graham@alt-three.com>
  */
 interface ApiInterface
```

Comment:
PR should be marked as BC breaking, because of this interface changes?

Reply:
Yes, this PR is breaking, and is made to the master branch, which is 3.x. There is no 3.0.0 release yet, so we are making breaking changes here on purpose. :)